src/Models/Traits/HasCustomAttributes.php

```diff
@@ -203,4 +203,9 @@ protected function throwMissingAttributeExceptionIfApplicable($key)
     {
         return $this->getCustomAttribute($key);
     }
+
+    protected function getAttributeFromArray($key)
+    {
+        return parent::getAttributeFromArray($key) ?? $this->getCustomAttribute($key);
+    }
```

Comment:
Is this to patch the internal function for `$value` to get the correct value?
What about calling `$this->getAttributeFromArray('price') ?? $this->getCustomAttribute('price')` manually?